bittensor/_dendrite/__init__.py

```diff
@@ -86,7 +90,12 @@ def __new__(
                 max_active_receptors = config.dendrite.max_active_receptors,
                 compression = config.dendrite.compression,
             )
-        if config.dendrite.multiprocessing:
+        if config.dendrite._mock:
+            return dendrite_impl.DendriteMock ( 
+                config = config,
+                wallet = wallet
+            )
+        elif config.dendrite.multiprocessing:
```

Comment:
We shouldn't have testing code inside of production code. creating mocks should be left to testing portions of the code, can we move this out?

Reply:
Lets redo the mocking but lets do it at a later date.